cmd/lightning-vault/main.go

```diff
@@ -98,6 +98,39 @@ func (h *Handlers) toLookup(data entities.Data, uniqueID string) {
 	h.Lookup[data.PubKey+uniqueID] = data
 }
 
+// NodeData struct.
+type NodeData struct {
+	UniqueID string
+	Data     entities.Data
+}
+
+func (h *Handlers) allNodes() <-chan NodeData {
+	glog.Info("All nodes")
+	ch := make(chan NodeData)
+
+	go func() {
+		defer close(ch)
+
+		for k, v := range h.Lookup {
+			if len(k) < utils.PUBKEY_LEN {
```

Comment:
when does it happen to be longer ?  Why cant we just use `if len(k) != PUBKEY_LEN` ?

Reply:
This is a bit of a legacy hack, but it is longer because it can either be in the form `pubkey` or `pubkey+uniqueid`.